File: GTTS/INL.TrainingService.Models/GetTrainingEventLocation_Item.cs
using System;

namespace INL.TrainingService.Models
{
    public class GetTrainingEventLocation_Item : IGetTrainingEventLocation_Item
    {
		public int? TrainingEventLocationID { get; set; }
		public int LocationID { get; set; }
		public DateTime EventStartDate { get; set; }
		public DateTime EventEndDate { get; set; }
		public DateTime? TravelStartDate { get; set; }
		public DateTime? TravelEndDate { get; set; }
        public int? ModifiedByAppUserID { get; set; }
		public DateTime? ModifiedDate { get; set; }
        public string LocationJSON { get; set; }
        public string ModifiedByUserJSON { get; set; }
        public string LocationName { get; set; }
		public string AddressLine1 { get; set; }
		public string AddressLine2 { get; set; }
		public string AddressLine3 { get; set; }
		public int CityID { get; set; }
		public string CityName { get; set; }
		public int StateID { get; set; }
		public string StateName { get; set; }
		public string StateCode { get; set; }
        public string StateAbbreviation { get; set; }
        public int CountryID { get; set; }
		public string CountryName { get; set; }
		public string CountryCode { get; set; }
        public string CountryAbbreviation { get; set; }

    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/get-training-event-location_item.ts


import { IGetTrainingEventLocation_Item } from './iget-training-event-location_item';

export class GetTrainingEventLocation_Item implements IGetTrainingEventLocation_Item {
  
	public TrainingEventLocationID?: number;
	public LocationID: number = 0;
	public EventStartDate: Date = new Date(0);
	public EventEndDate: Date = new Date(0);
	public TravelStartDate?: Date;
	public TravelEndDate?: Date;
	public ModifiedByAppUserID?: number;
	public ModifiedDate?: Date;
	public LocationJSON: string = "";
	public ModifiedByUserJSON: string = "";
	public LocationName: string = "";
	public AddressLine1: string = "";
	public AddressLine2: string = "";
	public AddressLine3: string = "";
	public CityID: number = 0;
	public CityName: string = "";
	public StateID: number = 0;
	public StateName: string = "";
	public StateCode: string = "";
	public StateAbbreviation: string = "";
	public CountryID: number = 0;
	public CountryName: string = "";
	public CountryCode: string = "";
	public CountryAbbreviation: string = "";
  
}






